framework_tvb/tvb/interfaces/web/controllers/users_controller.py

```diff
@@ -433,6 +434,11 @@ def _populate_version(self, template_dictionary):
         template_dictionary[KEY_CURRENT_VERSION_FULL] = TvbProfile.current.version.CURRENT_VERSION
         return template_dictionary
 
+    @staticmethod
+    def _set_base_url():
+        if not TvbProfile.current.web.BASE_URL:
+            TvbProfile.current.web.BASE_URL = cherrypy.request.base
```

Comment:
We should test this behind a Proxy.
Maybe OPenShift dev would be a good testing bed ?

Reply:
we leave this here, as it is used now only when sending emails